src/Symfony/Bridge/Doctrine/Tests/Fixtures/User.php

```diff
@@ -39,15 +39,15 @@ public function getRoles()
     {
     }
 
-    public function getPassword()
+    public function getPassword(): ?string
     {
     }
 
-    public function getSalt()
+    public function getSalt(): ?string
```

Comment:
We should add `return null;` both methods.

Reply:
that's not needed unless the methods are called - tests are green :)

Reply:
true